src/config/config.example.json

```diff
@@ -1,7 +1,8 @@
 {
-  "sitekey": "super secret key",
-  "GENERAL_API_URL": "http://localhost:8080",
-  "LOGGING_API_URL": "http://localhost:8081",
-  "MAILER_API_URL": "http://localhost:8082",
-  "RPC_API_URL": "http://localhost:8083"
-}
+    "sitekey": "super secret key",
+    "GENERAL_API_URL": "http://localhost:8080",
+    "LOGGING_API_URL": "http://localhost:8081",
+    "MAILER_API_URL": "http://localhost:8082",
+    "RPC_API_URL": "http://localhost:8083"
+  }
+  
```

Comment:
you can revert the changes in this file

Reply:
I am a little confused why it says I added these lines of code when this code is exactly the same as the dev branch